webpack/resolve-chunk-plugin/src/ResolveChunkPlugin.ts

```diff
@@ -1,76 +1,123 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
 import * as Webpack from 'webpack';
 
-// tslint:disable-next-line:variable-name
+// tslint:disable:variable-name
 const ConstDependency: IConstDependency = require('webpack/lib/dependencies/ConstDependency');
+const ParserHelpers: IParserHelper = require('webpack/lib/ParserHelpers');
+// tslint:enable:variable-name
+
+const EXPRESSION_NAME: string = 'resolveChunk';
+const PLUGIN_NAME: string = 'resolve-chunk-plugin';
 
 /**
  * See README for documentation on this plugin.
  *
  * @public
  */
 export class ResolveChunkPlugin implements Webpack.Plugin {
+  private _chunkIdMap: Map<string, ((id: number | undefined) => void)[]> =
+    new Map<string, ((id: number | undefined) => void)[]>();
```

Comment:
>(id: number | undefined) => void)[] [](start = 21, length = 35)

This expression is unwieldy.  `(id: number | undefined) => void)` should be given a name